docs/scripting.md

```diff
@@ -493,6 +493,16 @@ Using previous examples:
 
 For the second example, it's worth thinking about what messages the user would see. If you have an error handler that replies to the user, you may not need to add a custom message and could send back the error message provided to the `get()` request, but of course it depends on how public you want to be with your exception reporting.
 
+## Logging
+Hubot uses [log](https://www.npmjs.com/package/log) to manage logging.
+You can set log level and file via `HUBOT_LOG_LEVEL` and `HUBOT_LOG_FILE` configs, or `--log-level` and `--log-file` as command line options.
+
+```coffeescript
+robot.logger.debug('this way!')
```

Comment:
I feel like there is room for a little more explanation and example here to better match with the style throughout the document (set up a scenario and show how this would get used). This makes it easier for less-experienced developers to understand how to use the functions.